lib/workers/repository/update/branch/get-updated.ts

```diff
@@ -334,10 +337,7 @@ export async function getUpdatedPackageFiles(
       if (updateArtifacts) {
         const packageFileContents =
           updatedFileContents[packageFile] ||
-          (await getFile(
-            packageFile,
-            reuseExistingBranch ? config.branchName : config.baseBranch
-          ));
+          (await getFile(packageFile, config.baseBranch));
```

Comment:
reuse existing is always false here?

Reply:
Yes, this code is inside a if block which confirms that `reuseExistingBranch` is `false`.